.github/workflows/docker-publish.yml

```diff
@@ -81,4 +90,60 @@ jobs:
           COSIGN_EXPERIMENTAL: "true"
         # This step uses the identity token to provision an ephemeral certificate
         # against the sigstore community Fulcio instance.
-        run: echo "${{ steps.meta.outputs.tags }}" | xargs -I {} cosign sign {}@${{ steps.build-and-push.outputs.digest }}
+        run: |
+          echo "${{ env.VERSION }}" | xargs -I {} cosign sign {}@${{ steps.build-and-push.outputs.digest }}
```

Comment:
Why we need to replace `steps.meta.outputs.tags` with `env.VERSION`?

Reply:
I was doing some experiment with it, and I left it forgotten, fixed now :)